_data/wgs.yml

```diff
@@ -72,77 +74,80 @@
   bottomliner: Agost Biro
   slug: business-models
   status: Incubating
-  ranking: 1
+  ranking: 0
   description: "For individual developers, what does sustainability look like? How can developers get money for their projects, taking them out of the abandon-ware and hobbyist stage?"
 
 - title: Membership models for OSS
   bottomliner: Gabriela Rodríguez Berón
   slug: membership-models
   status: Incubating
-  ranking: 1
+  ranking: 0
   description: "What do models of membership look like, specifically for organisations developing FLOSS? How can we all be involved?"
 
 - title: Climate Emergency
   bottomliner: Xavier Damman
   slug: climate-emergency
   status: Incubating
-  ranking: 1
+  ranking: 0
   description: "Sustainability doesn't just mean code. How can we make our work sustainable environmentally, too? How can we share our knowledge of open source with other groups in the world?"
 
 - title: Community Principles & Ostrom Redux
   bottomliner: Greg Bloom
   slug: ostrom
-  status: Active
-  ranking: 2
+  status: Incubating
+  ranking: 0
   description: "Elinor Ostrom laid out principles of healthy communities for governing the commons. How can we apply her Nobel-prize winning research to the world of open source?"
 
 - title: Governance Readiness
   bottomliner: Javier Izquierdo
   slug: governance-readiness
-  status: Incubating
-  ranking: 0
+  url: true
+  status: Active
+  ranking: 1
   description: "What does healhty governance look like? How can we improve our day-to-day work, identify barriers and needs, and intervene where new governance structures are needed?"
 
 - title: Marketing
   bottomliner: Richard Littauer
   slug: marketing
   status: Incubating
-  ranking: 1
+  ranking: 0
   description: "What does an open source developer have to do to find new users, more sponsors, and better swag? What is sustainable marketing for open source?"
 
 - title: Principles of Authentic Participation
   bottomliner: Justin W. Flory
   slug: authentic-participation
   url: true
   status: Active
-  ranking: 2
+  ranking: 1
   description: "What are the core set of principles for what authentic participation means in multiple contexts (e.g. corporate, individual, humanitarian/NGOs, etc.), as it relates to OSS code?"
 
 - title: Transparency Goals
   bottomliner: Duane O'Brien
   slug: transparency-goals
-  status: Active
-  ranking: 2
+  status: Incubating
+  ranking: 0
   description: "Good corp, bad corp: what does sustainability and ethical practices mean for corporations who run OSPOs or who use open source code?"
```

Comment:
Not sure if @DuaneOBrien agrees, but the Principles of Authentic Participation working group came from Duane's Transparency Goals sessions at Sustain. Maybe this is in the direction of accountability though, which we haven't explored in depth on P.A.P. working group.

Reply:
This is a different group. I'm waiting for Duane's PR. I wouldn't touch it, for now.